go/client/cmd_wallet_send.go

```diff
@@ -29,6 +30,10 @@ func newCmdWalletSend(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Co
 			Name:  "m, message",
 			Usage: "Include a message with the payment.",
 		},
+		cli.StringFlag{
+			Name:  "f, from-account-id",
```

Comment:
`-f` is often force, which we might want since this currently asks for confirmation.

How about just `from`?